src/components/CalendarBodyForMonthView.tsx

```diff
@@ -66,9 +75,11 @@ function _CalendarBodyForMonthView<T extends ICalendarEventBase>({
   onPressMoreLabel,
   sortedMonthView,
   renderCustomDateForMonth,
+  disableMonthEventCellPress,
 }: CalendarBodyForMonthViewProps<T>) {
   const { now } = useNow(!hideNowIndicator)
   const [calendarWidth, setCalendarWidth] = React.useState<number>(0)
+  const [calendarCellHeight, setCalendarCellHeight] = React.useState<number>(0)
```

Comment:
In IOS, it seems that `height: 100%` does not include the parent's padding, which leads to child components to be smaller than parent components. 
So I added this state and set the height manually through onLayout

Reply:
Thanks!